src/param.rs

```diff
@@ -626,24 +625,42 @@ impl AudioParamProcessor {
     }
 
     fn mix_to_output(&mut self, input: &AudioRenderQuantum, output: &mut AudioRenderQuantum) {
-        #[cfg(test)]
-        assert_eq!(self.buffer.len(), RENDER_QUANTUM_SIZE);
```

Comment:
I still like this test assertion, maybe bring it back with `len == 1 or RENDER_QUANTUM_SIZE`
It might guard us in the future for wrong boundary checks for filling the buffer

Reply:
done